system/Config/Factories.php

```diff
@@ -145,7 +145,7 @@ public static function __callStatic(string $component, array $arguments)
             }
 
             // Try to locate the class
-            if ($class = self::locateClass($options, $alias)) {
+            if (($class = self::locateClass($options, $alias)) !== null && ($class = self::locateClass($options, $alias)) !== '') {
```

Comment:
The Psalm error below seems to be a false positive.
But the changed code is difficult to read and it calls `self::locateClass()` twice.

```suggestion
            $class = self::locateClass($options, $alias)
            if ($class !== null) {
```